/* Fixing a bug with thunder, the sound of thunder and flashes on the level of Hatshepsut. */
--- a/SamTFE/Sources/Entities/Lightning.es	2023-02-08 19:14:42.461910992 +0300
+++ b/SamTFE/Sources/Entities/Lightning.es	2023-02-09 10:07:51.374219596 +0300
@@ -118,9 +118,7 @@
     }
 
     // wait for sound to progress to lightning strike
-    if (_atiThunderSounds[ m_iSoundPlaying].ti_fThunderStrikeDelay>0.0f) {
     autowait( _atiThunderSounds[ m_iSoundPlaying].ti_fThunderStrikeDelay);
-    }
 
     // remember current time as lightning start time
     TIME tmNow = _pTimer->CurrentTick();

